examples/napi/src/class.rs

```diff
@@ -85,6 +85,23 @@ impl Animal {
   pub fn return_other_class_with_custom_constructor(&self) -> Bird {
     Bird::new("parrot".to_owned())
   }
+
+  #[napi]
+  pub fn override_individual_arg_on_method(
+    &self,
+    normal_ty: String,
+    #[napi(ts_arg_type = "string")] overridden_ty: napi::JsObject,
```

Comment:
maybe it'd be better to override this with something that's a sub-type of object

Reply:
Switched it to this ` #[napi(ts_arg_type = "{n: string}")] overridden_ty: napi::JsObject,` to be like an anonymous type.